### PR TITLE
Fix Modal causing scrollbar to disappear

### LINKED ISSUE #1103: Modal windows cause the topNav layout change in fixed topNav mode
Watch the top right corner of this gif:
modal-scrollboar-glitch

It's a minor issue though. Happens only when the topNav is in fixed mode.

### PR DESCRIPTION
**Pull request by FergusMok**
**What is the purpose of this pull request?**
Fixes #1103 

- [ ] Documentation update
- [X] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

**Overview of changes:**
Trivial CSS fix to allow the modal to open without hiding the scrollbar. 

**Anything you'd like to highlight / discuss:**

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Fix modal causing scrollbar to disappear 

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [X] Linked all related issues
- [ ] No unrelated changes

### CODE REVIEW

**Review by jonahtanjz — COMMENTED** [review]
Thank you for your contribution @FergusMok 👍 

I have left a comment for the change :)

File: packages/core-web/src/styles/markbind.css [hunk 1]
```diff
@@ -243,6 +243,10 @@ li.footnote-item:target {
     opacity: 1;
 }
 
+.modal-open {
+    overflow: scroll;
```
[the remark this answers is not in the document]
Comment (FergusMok) [reply packages/core-web/src/styles/markbind.css]: Hi Jonah, yup that'll be much better. I did not consider this case previously. I have made the relevant changes, thanks alot! 👍

**Review by jonahtanjz — COMMENTED** [review]
Thanks for making the change!

After thoroughly testing this out, I noticed that this fix introduces a few other issues.

For instance, when the modal is opened, the page can now be scrolled and when the modal closes, it causes it to "jump".

https://user-images.githubusercontent.com/47470981/146667110-2c25ff14-03d7-4989-a173-ab2a26a8216e.mp4

Perhaps we can "block" the user from scrolling the page when the modal is opened but still show the scroll bar?

The other issue I found is that when the content of the modal is scrollable, it adds another scroll bar, leading to 2 scroll bars.

image

For this, the fix I can think of right now is to remove the `overflow: auto` css if the content of the modal is scrollable.

@FergusMok feel free to suggest alternatives if you can come up with simpler solutions :)

**Comment by ang-zeyu** [comment]
Closing this as its been stale for a while, the issue is still open.

**Pull request closed without merging**